configuration/set.go

```diff
@@ -25,18 +25,28 @@ import (
 	"path/filepath"
 	"strings"
 
-	. "github.com/sonatype-nexus-community/nancy/logger"
+	"github.com/sirupsen/logrus"
+	"github.com/sonatype-nexus-community/nancy/logger"
 	"github.com/sonatype-nexus-community/nancy/types"
 	"gopkg.in/yaml.v2"
 )
 
+// these consts must match their associated yaml tag below. for use where tag name matters, like viper
+const YamlKeyIQServer = "IQServer"
+const YamlKeyIQUsername = "IQUsername"
+const YamlKeyIQToken = "IQToken"
+
 // IQConfig is a struct for holding IQ Configuration, and for writing it to yaml
 type IQConfig struct {
-	Server   string `yaml:"Server"`
-	Username string `yaml:"Username"`
-	Token    string `yaml:"Token"`
+	IQServer   string `yaml:"IQServer"`
+	IQUsername string `yaml:"IQUsername"`
+	IQToken    string `yaml:"IQToken"`
```

Comment:
Why did we need the extra specificity here?? The yaml for iQ is already in its own directory right?

`ConfigLocation = filepath.Join(HomeDir, types.IQServerDirName, types.IQServerConfigFileName)`